libs/ui/layout/src/lib/value-list/value-list.component.html

```diff
@@ -0,0 +1,44 @@
+<div
+  class="flex items-center justify-center"
+  cdkOverlayOrigin
+  #trigger="cdkOverlayOrigin"
+  [class]="extraClass"
+>
+  <gn-ui-button
+    #iconElement
+    type="light"
+    extraClass="flex items-center bg-transparent border-none justify-center"
+    (buttonClick)="toggleOverlay()"
+  >
+    <ng-icon name="iconoirList" size="24"></ng-icon>
+  </gn-ui-button>
+
+  <ng-template
+    cdkConnectedOverlay
+    [cdkConnectedOverlayOrigin]="trigger"
+    [cdkConnectedOverlayOpen]="isOpen"
+    [cdkConnectedOverlayPositions]="[overlayPosition]"
+    cdkConnectedOverlayPush
+    [cdkConnectedOverlayWidth]="'auto'"
+    [cdkConnectedOverlayFlexibleDimensions]="true"
+    [cdkConnectedOverlayGrowAfterOpen]="true"
+    (detach)="close()"
+  >
+    <div class="bg-white shadow-lg border border-gray-300 flex">
+      <ul
+        class="list-disc list-inside m-2 space-y-1 w-64 max-h-60 overflow-y-auto"
+      >
+        <li *ngFor="let v of values">{{ v.label || v.code }}</li>
```

Comment:
Floating overlay even when scrolling, should implement the same behavior as for the long text cell (keep sticky, close overlay when opening another seciton...)
![image](https://github.com/user-attachments/assets/57725940-d751-45fd-9fb4-3697b7d3f067)